src/server.js

```diff
@@ -15,23 +15,23 @@ let express = require('express'),
 
 // Load configurations
 // if test env, load example file
-let env = process.env.NODE_ENV = process.env.NODE_ENV || 'development',
+const env = process.env.NODE_ENV = process.env.NODE_ENV || 'development', // eslint-disable-line
```

Comment:
const env = process.env.NODE_ENV || 'development' and remove eslint-disable

Reply:
When I changed this, there was that resulted. Perhaps we could examine why that happened?